apps/web/src/app/(signing)/sign/[token]/date-field.tsx

```diff
@@ -35,12 +43,20 @@ export const DateField = ({ field, recipient }: DateFieldProps) => {
 
   const isLoading = isSignFieldWithTokenLoading || isRemoveSignedFieldWithTokenLoading || isPending;
 
+  const localDateFormattedValue = convertToLocalSystemFormat(
+    field.customText,
+    dateFormat,
+    timezone,
+  );
+
+  const dateFormatValue = dateFormat === null ? 'yyyy-MM-dd hh:mm a' : dateFormat;
```

Comment:
If we had this a constant you could also do `dateFormat ?? DEFAULT_DOCUMENT_DATE_FORMAT` here

Reply:
Done Cap'n